### DISCUSSION TITLE
Ubuntu LXC Script Failure

### DISCUSSION THREAD

**Discussion by Photo64** [tteck/Proxmox#797] [Help Needed] [answered]
I ran this script -> bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/ubuntu-v4.sh)".  to install ubuntu however it appears as though it failed to complete. It looks like the container is running however I am not sure how to access it. Should the creation of the container completed more steps prior to the failure?

Using Default Settings
Using ubuntu Version: 22.04
Using Container Type: Unprivileged NO DEVICE PASSTHROUGH
Using Root Password: Automatic Login
Using Container ID: 100
Using Hostname: ubuntu
Using Disk Size: 2GB
Allocated Cores 1
Allocated Ram 512
Using Bridge: vmbr0
Using Static IP Address: dhcp
Using Gateway Address: Default
Using MAC Address: Default
Using VLAN Tag: Default
Enable Root SSH Access: No
Creating a Ubuntu LXC using the above default settings
 ✓ Using local for Template Storage.
 ✓ Using local-lvm for Container Storage.
 ✓ Updated LXC Template List
 ✓ Downloaded LXC Template
 ✓ LXC Container 100 was successfully created.
 ✓ Started LXC Container
 ✓ Set up Container OS
 ✓ Network Connected: 192.168.1.61 
 ✓ Internet Connected
 ✓ DNS Resolved github.com to 140.82.112.4
 - Updating Container OS...‼ ERROR 100@79 Unknown failure occurred.

**Comment by tteck** [comment]
When it finishes you'll get `✓ Completed Successfully!`
Remove and start over
> `‼ ERROR 100@79 Unknown failure occurred.`

https://github.com/tteck/Proxmox/blob/a42bdbee40b19d265f72d9ce046fde6cc844471e/setup/ubuntu-install.sh#L79

**Comment by Photo64** [comment]
Failed again
 - Updating Container OS...‼ ERROR 100@80 Unknown failure occurred.  
 
Looks like at the next line?  Again it appears the container is running.

**Reply by tteck** [reply]
The container is running, but using autologin you can't do nothing.
> `‼ ERROR 100@80 Unknown failure occurred`

https://github.com/tteck/Proxmox/blob/a42bdbee40b19d265f72d9ce046fde6cc844471e/setup/ubuntu-install.sh#L80

**Reply by tteck** [reply]
Use "Advanced" and set a password if you want to continue with the running container.

**Comment by Photo64** [comment]
I guess third time was the charm.  IS there a port# for access?

Creating a Ubuntu LXC using the above default settings
 ✓ Using local for Template Storage.
 ✓ Using local-lvm for Container Storage.
 ✓ Updated LXC Template List
 ✓ LXC Container 100 was successfully created.
 ✓ Started LXC Container
 ✓ Set up Container OS
 ✓ Network Connected: 192.168.1.68 
 ✓ Internet Connected
 ✓ DNS Resolved github.com to 140.82.113.4
 ✓ Updated Container OS
 ✓ Installed Dependencies
 ✓ Customized Container
 ✓ Cleaned
 ✓ Completed Successfully!

**Reply by tteck** [reply]
No, it's headless

**Reply by tteck** [reply]
Console access

**Comment by Photo64** [comment, marked as answer]
Could I ask another question -> Trying to SSH into the container and I tried running this -> echo "PermitRootLogin yes" > /etc/ssh/sshd_config.d/rootlogin.conf && systemctl restart sshd.service.   however I am still unable to connect. Would you know if there is something else I need to do?

**Reply by tteck** [reply]
https://github.com/tteck/Proxmox/discussions/385
